circle.yml

```diff
@@ -194,8 +194,8 @@ commands:
       browser:
         description: Name of the browser to use
         type: enum
-        enum: ["electron", "chrome", "firefox"]
-        default: "electron"
+        enum: ["", "electron", "chrome", "firefox"]
```

Comment:
Can you explain why the empty enum value is necessary?

Reply:
having default browser ALWAYS appended `--browser electron` which breaks some NPM scripts (the ones that use `start-server-and-test`), so had to find a way to NOT have it